src/main/java/twilightforest/entity/EntityTFYeti.java

```diff
@@ -17,202 +18,213 @@
-	}
-
-	public void setAngry(boolean anger) {
-		dataManager.set(ANGER_FLAG, anger);
-
-		if (!world.isRemote) {
-			if (anger) {
-				if (!getAttribute(SharedMonsterAttributes.FOLLOW_RANGE).hasModifier(ANGRY_MODIFIER)) {
-					this.getAttribute(SharedMonsterAttributes.FOLLOW_RANGE).applyModifier(ANGRY_MODIFIER);
-				}
-			} else {
-				this.getAttribute(SharedMonsterAttributes.FOLLOW_RANGE).removeModifier(ANGRY_MODIFIER);
-			}
-		}
-	}
-
-	@Override
-	public void writeAdditional(CompoundNBT compound) {
-		super.writeAdditional(compound);
-		compound.putBoolean("Angry", this.isAngry());
-	}
-
-	@Override
-	public void readAdditional(CompoundNBT compound) {
-		super.readAdditional(compound);
-		this.setAngry(compound.getBoolean("Angry"));
-	}
-
-	/**
-	 * Put the player out in front of where we are
-	 */
-	@Override
-	public void updatePassenger(Entity passenger) {
-		Vec3d riderPos = this.getRiderPosition(passenger);
-		passenger.setPosition(riderPos.x, riderPos.y, riderPos.z);
-	}
-
-	/**
-	 * Returns the Y offset from the entity's position for any entity riding this one.
-	 */
-	@Override
-	public double getMountedYOffset() {
-		return 2.25D;
-	}
-
-	/**
-	 * Used to both get a rider position and to push out of blocks
-	 */
-	private Vec3d getRiderPosition(@Nullable Entity passenger) {
-		if (passenger != null) {
-			float distance = 0.4F;
-
-			double dx = Math.cos((this.rotationYaw + 90) * Math.PI / 180.0D) * distance;
-			double dz = Math.sin((this.rotationYaw + 90) * Math.PI / 180.0D) * distance;
-
-			return new Vec3d(this.getX() + dx, this.getY() + this.getMountedYOffset() + passenger.getYOffset(), this.getZ() + dz);
-		} else {
-			return new Vec3d(this.getX(), this.getY(), this.getZ());
-		}
-	}
-
-	@Override
-	public boolean canRiderInteract() {
-		return true;
-	}
-
-	@Override
-	public boolean getCanSpawnHere() {
-		// don't check light level in the snow
-		if (world.getBiome(new BlockPos(this)) == TFBiomes.snowy_forest.get()) {
-			return world.checkNoEntityCollision(this) && world.getCollisionBoxes(this, getBoundingBox()).size() == 0;
-		} else {
-			// normal EntityMob spawn check, checks light level
-			return super.getCanSpawnHere();
-		}
-	}
-
-	@Override
-	protected boolean isValidLightLevel() {
-		return world.getBiome(new BlockPos(this)) == TFBiomes.snowy_forest.get() || super.isValidLightLevel();
-	}
-
-	@Override
-	protected float getSoundPitch() {
-		return super.getSoundPitch() + 0.55F;
-	}
-
-	@Nullable
-	@Override
-	protected SoundEvent getAmbientSound() {
-		return TFSounds.ALPHAYETI_GROWL;
-	}
-
-	@Override
-	protected SoundEvent getHurtSound(DamageSource damageSourceIn) {
-		return TFSounds.ALPHAYETI_HURT;
-	}
-
-	@Override
-	protected SoundEvent getDeathSound() {
-		return TFSounds.ALPHAYETI_DIE;
-	}
-
-	@Override
-	public ResourceLocation getLootTable() {
-		return LOOT_TABLE;
-	}
+    public static final ResourceLocation LOOT_TABLE = TwilightForestMod.prefix("entities/yeti");
```

Comment:
Class is still entirely changed due to incorrect spacing. Note that the diff says the content was deleted and then added back in.